pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -239,17 +250,22 @@ def solve(self, model, **kwds):
         solve_data.results = SolverResults()
         solve_data.timing = Container()
 
+        solve_data.original_model = model
+        solve_data.working_model = model.clone()
+        if config.integer_to_binary:
+            TransformationFactory('contrib.integer_to_binary'). \
+                apply_to(solve_data.working_model)
```

Comment:
I just want to point out the reason it was necessary to move the `working_model` declaration further up (outside of the `create_utility_block` context) is that the `TransformationFactory('contrib.integer_to_binary')` introduces new variables. This way `solve_data.working_model.MindtPy_utils.variable_list` will include those variables.